bcdevexchange/wwwroot/css/style.css

```diff
@@ -27,50 +27,47 @@ body {
   font-weight: normal;
   font-style: normal;
   overflow-x: hidden;
+  color: #fff;
 }
 
-
 img {
   max-width: 100%;
   width: 100%;
   height: auto;
 }
 
-a,
-.button {
+a, .button {
   -webkit-transition: 0.3s;
   -moz-transition: 0.3s;
   -o-transition: 0.3s;
   transition: 0.3s;
 }
 
+a {
+  color: #fff;
+}
+
 a:focus,
 .button:focus, button:focus {
   text-decoration: none;
   outline: none;
 }
 
 a:focus {
-  color: #0065f2;
+  color: #fff;

```

Comment:
Why is this being changed?

Reply:
I did this so that the inline linked text (e.g. "Request an invitation" - see line 131 in Index.cshtml) doesn't turn blue.